guide/sequelize/README.md

```diff
@@ -172,17 +172,17 @@ catch (e) {
 }
 ```
 
-`Tags.create()` uses our models that we created previously. The `.create()` method inserts some data into the model. We're going to be inserting a tag name, description, and the author name into the database.  
-`catch (e)` This section is absolutely necessary for our insert. We offload checking for duplicates to the database, so that it will tell us if we create a tag that already exists. The alternative is to query the database before adding data, and checking if we get a result. If we don't, only then do we add the data. But this requires two queries instead of one, so this method is less work.   
-`if (e.name === "SequelizeUniqueConstraintError")` Although this was mostly for doing less work, it's always good to handle your errors, especially if you know what types of errors you will receive. This error comes up if your unique constraint is violated, i.e. someone inserted duplicate values.
+`Tags.create()` uses the models that was created previously. The `.create()` method inserts some data into the model. you are going to be inserting a tag name, description, and the author name into the database.  
```

Comment:
Again, should not be passively be created, the user did it
You capital after `.`
"going to insert" is much cleaner to read.
```suggestion
`Tags.create()` uses the models that you created previously. The `.create()` method inserts some data into the model. You are going to insert a tag name, description, and the author name into the database.  
```